broker/src/test/java/io/zeebe/broker/system/configuration/RocksdbCfgTest.java

```diff
@@ -71,8 +72,16 @@ public void shouldCreateRocksDbConfigurationFromDefault() {
     assertThat(rocksDbConfiguration.getColumnFamilyOptions()).isEmpty();
     assertThat(rocksDbConfiguration.isStatisticsEnabled()).isFalse();
     assertThat(rocksDbConfiguration.getMemoryLimit())
-        .isEqualTo(DataSize.ofMegabytes(512).toBytes());
-    assertThat(rocksDbConfiguration.getMaxOpenFiles()).isEqualTo(-1);
+        .isEqualTo(RocksDbConfiguration.DEFAULT_MEMORY_LIMIT);
+    assertThat(rocksDbConfiguration.getMaxOpenFiles())
+        .isEqualTo(RocksDbConfiguration.DEFAULT_UNLIMITED_MAX_OPEN_FILES);
+    assertThat(rocksDbConfiguration.getMaxWriteBufferNumber())
+        .isEqualTo(RocksDbConfiguration.DEFAULT_MAX_WRITE_BUFFER_NUMBER);
+    assertThat(rocksDbConfiguration.getMinWriteBufferNumberToMerge())
+        .isEqualTo(RocksDbConfiguration.DEFAULT_MIN_WRITE_BUFFER_NUMBER_TO_MERGE);
+    assertThat(rocksDbConfiguration.getIoRateBytesPerSecond())
+        .isEqualTo(RocksDbConfiguration.DEFAULT_IO_RATE_BYTES_PER_SECOND);
+    assertThat(rocksDbConfiguration.isWalDisabled()).isFalse();
```

Comment:
What I learned in the past is that sometimes you dont want to use the static defaults and more here magic numbers to realize if someone changes defaults. Otherwise the tests will be still green. IT is discussable of course.

Reply:
Hm, I understood we wanted to make sure these are set to the defaults always. But I see your point, it makes sense and we should be aware when these change.